internal/admission/sts.go

```diff
@@ -0,0 +1,75 @@
+package admission
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	appsv1 "k8s.io/api/apps/v1"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+type STSHandler struct {
+	decoder *admission.Decoder
```

Comment:
There is a`DefaultHandler` in `pkg/admission/handlers` that can be used to inherit methods for the interface.